<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterControl</class>
 <widget class="QMainWindow" name="MasterControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MasterControl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>287</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="openPushButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="distanceSpinBox">
       <property name="minimum">
        <number>-1000000</number>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="speedSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Distance(absolute):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baud rate:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Serial Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudrateComboBox">
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="movePushButton">
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="serialNumberComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lightLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:12px;
background-color:rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshSerialNumber">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshSerialModbusNumber">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="lightModbusLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:12px;
background-color:rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>278</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QPushButton" name="openModbusPushButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Weight Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Baud rate:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Serial Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudrateModbusComboBox">
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="weightReadPushButton">
       <property name="text">
        <string>Read Weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="serialNumberModbusComboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="weightTypeComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>It's no use the following, need more infomation.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dof Control</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
